src/main/java/sirius/kernel/xml/Outcall.java

```diff
@@ -325,6 +334,14 @@ public void setReadTimeout(int timeoutMillis) {
         modifyRequest().timeout(Duration.ofMillis(timeoutMillis));
     }
 
+    /**
+     * Sets the connect and read timeout to a interactively reasonable duration defined in http.outcall.interactive.*
+     */
+    public void setInteractive() {
```

Comment:
Never call a method setXXX unless it is a setter. Also, why not simply provide a proper timeout here instead of calling a magic method....

Reply:
Renamed and added soap config plus call in SoapClient as discussed